sensing/imu_corrector/src/imu_corrector_core.cpp

```diff
@@ -40,19 +74,47 @@ void ImuCorrector::callbackImu(const sensor_msgs::msg::Imu::ConstSharedPtr imu_m
   sensor_msgs::msg::Imu imu_msg;
   imu_msg = *imu_msg_ptr;
 
-  imu_msg.angular_velocity.x -= angular_velocity_offset_x_;
-  imu_msg.angular_velocity.y -= angular_velocity_offset_y_;
-  imu_msg.angular_velocity.z -= angular_velocity_offset_z_;
+  imu_msg.angular_velocity.x -= angular_velocity_offset_x_imu_link_;
+  imu_msg.angular_velocity.y -= angular_velocity_offset_y_imu_link_;
+  imu_msg.angular_velocity.z -= angular_velocity_offset_z_imu_link_;
+
+  imu_msg.angular_velocity_covariance[COV_IDX::X_X] =
+    angular_velocity_stddev_xx_imu_link_ * angular_velocity_stddev_xx_imu_link_;
+  imu_msg.angular_velocity_covariance[COV_IDX::Y_Y] =
+    angular_velocity_stddev_yy_imu_link_ * angular_velocity_stddev_yy_imu_link_;
+  imu_msg.angular_velocity_covariance[COV_IDX::Z_Z] =
+    angular_velocity_stddev_zz_imu_link_ * angular_velocity_stddev_zz_imu_link_;
+  imu_msg.linear_acceleration_covariance[COV_IDX::X_X] =
+    accel_stddev_imu_link_ * accel_stddev_imu_link_;
+  imu_msg.linear_acceleration_covariance[COV_IDX::Y_Y] =
+    accel_stddev_imu_link_ * accel_stddev_imu_link_;
+  imu_msg.linear_acceleration_covariance[COV_IDX::Z_Z] =
+    accel_stddev_imu_link_ * accel_stddev_imu_link_;
+
+  geometry_msgs::msg::TransformStamped::ConstSharedPtr tf_base2imu_ptr;
+  try {
+    tf_base2imu_ptr =
+      transform_listener_->getLatestTransform(output_frame_, imu_msg.header.frame_id);
+  } catch (tf2::TransformException & ex) {
```

Comment:
I think it will not work because it is already try-catch [here](https://github.com/autowarefoundation/autoware.universe/blob/main/common/tier4_autoware_utils/include/tier4_autoware_utils/ros/transform_listener.hpp#L48-L54) .

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/2360/commits/a1d43f626ff132b57ed8895d37cb1020fb99a3e2
Resolved